pkg/config/tls.go

```diff
@@ -2,22 +2,23 @@ package config
 
 import "fmt"
 
-func InClusterServiceName() string {
-	return KyvernoServiceName() + "." + KyvernoNamespace() + ".svc"
+func InClusterServiceName(commonName string, namespace string) string {
+	return commonName + "." + namespace + ".svc"
 }
 
-func DnsNames() []string {
+func DnsNames(commonName string, namespace string) []string {
 	return []string{
-		KyvernoServiceName(),
-		fmt.Sprintf("%s.%s", KyvernoServiceName(), KyvernoNamespace()),
-		InClusterServiceName(),
+		commonName,
+		"svc." + commonName,
```

Comment:
What is this new pattern?

Reply:
It is this pattern: https://github.com/nirmata/kyverno-notation-aws/blob/4bbad6707201379ddf2a0092de0046854a5d9ab1/configs/samples/kyverno-policy.yaml#L47
```yaml
url: https://svc.kyverno-notation-aws/checkimages
```

Reply:
How does that work ?